specs/activity-api.yaml

```diff
@@ -596,11 +752,15 @@ components:
     ProviderEvent:
```

Comment:
the event may optionally contain a `requestorPubKey` field (hex encoded string)

(it's my fault for not adding it in a previous PR)

Reply:
my mistake - it's already there